### ISSUE TITLE
feature_localansible - SSH Connection Failing

### ISSUE THREAD

**Issue by LopezNathan** [LopezNathan/vpn-deployer#19]
The initial SSH connection to the droplet is failing when attempting to execute the playbook remotely : 

```
fatal: [droplet1]: UNREACHABLE! => {
    "changed": false,
    "msg": "Failed to connect to the host via ssh: ssh: connect to host IP port 22: Connection timed out",
    "unreachable": true
}
```

Although, if you utilize the same connection string used by Ansible and connect manually, it works and after you can successfully re-run the playbook with no issues.

I believe this is related to Ansible unable to add to the list of known hosts?

**Comment by LopezNathan** [comment]
This looks to be covered by Ansible here : https://docs.ansible.com/ansible/latest/user_guide/intro_getting_started.html#host-key-checking

**Comment by LopezNathan** [comment]
Temporarily fixed - 45a49a7

This was due to the Droplet still initializing during the deploy playbook execution, resulting in the SSH connection failing. If the process waits 10-20 seconds the issue does not occur. Temporarily fixed with a sleep but polling should be used instead.
